### PR TITLE
Add appvyor config

### PR DESCRIPTION
**Pull request by CAMOBAP795**
- https://github.com/metanorma/metanorma-standoc/issues/55

### CODE REVIEW

**Comment by ronaldtse** [comment]
@CAMOBAP795 all tests filed here.

**Comment by CAMOBAP795** [comment]
@ronaldtse yep I will have a look

**Comment by opoudjis** [comment]
There's two issues in Appveyor:

1. Neither Appveyor nor Travis are recognising the request to have three levels of Table of Contents. That is a new issue, which will likely go away when I do the next release of the gems. Ignore it.
2. Appveyor, unlike Travis, is failing to convert LaTeX to MathML, which means it is not finding the latex2mathmnl tool. That, @CAMOBAP795, you should investigate.

**Comment by ronaldtse** [comment]
@camobap795 the latex2mathml is a gem dependency of Metanorma (it’s one of our gems) maybe it’s not getting installed.

**Comment by CAMOBAP795** [comment]
@ronaldtse  Updated PR and also
 - https://github.com/metanorma/chocolatey-metanorma/pull/22

**Comment by CAMOBAP795** [comment]
still didn't work as expected, I will check

**Comment by CAMOBAP795** [comment]
latexmlmath didn't provide expected output, maybe because of input encoding UTF-8, Investigation in progress

**Comment by CAMOBAP795** [comment]
Encoding fixed with `chcp 65001` but I still reported https://github.com/brucemiller/LaTeXML/issues/1111

**Comment by CAMOBAP795** [comment]
@ronaldtse @opoudjis now there is left error which is the same as on travis, so I think we are ready to merge

File: in-test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+M = \begin{bmatrix} -\sin \lambda_0 & \cos \lambda_0 & 0 \\ -\sin \varphi_0 \cos \lambda_0 & -\sin \varphi_0 \sin \lambda_0 & \cos \varphi_0 \\ \cos \varphi_0 \cos \lambda_0 & \cos \varphi_0 \sin \lambda_0 & \sin \varphi_0 \end{bmatrix}
```
Comment (ronaldtse) [inline in-test.txt]: @CAMOBAP795 Is this a new file?

Comment (CAMOBAP795) [reply in-test.txt]: Added by mistake, I will remove it

**Comment by opoudjis** [comment]
Happy to merge once Appveyor passes.

**Comment by opoudjis** [comment]
And @CAMOBAP795, thank you for your outstanding work!

**Pull request merged**